doc/kubernetes/modules/ROOT/pages/running/keycloak-with-external-infinispan-deployment.adoc

```diff
@@ -0,0 +1,60 @@
+= Keycloak with external Infinispan Deployment
+:description: This describes configuration tweaks and changes on how to run Keycloak with an external Infinispan Deployment under load.
+
+{description}
+
+== Audience
+
+This guide describes advanced Infinispan configurations for Keycloak on Kubernetes which are load tested and will recover from single Pod failures.
+
+While the Helm charts in the Keycloak Benchmark project mix different aspects of production style deployments with instrumentation and monitoring, this documentation focuses on a minimal deployment with optional add-ons which admins can opt in for their own deployments.
+
+See xref:running/index.adoc[] for additional guides.
+
+== Prerequisites
+
+* OpenShift or Kubernetes cluster running.
+* Understanding of the xref:running/keycloak-deployment.adoc[Basic Keycloak deployment].
+* Understanding of a https://infinispan.org/docs/helm-chart/main/helm-chart.html[Infinispan deployment using Helm charts].
+
+== Procedure
+
+* Prepare an Infinispan Cache configuration XML, which would then be consumed in the Keycloak CR.
+* Create Config Map, Secret and other Kubernetes resources needed for Keycloak to connect to an external Infinispan deployment.
+* Extend the Keycloak Custom Resource with below Kubernetes deployment resources.
+
+=== Remote store username and password defined in a Kubernetes Secret
+[source,yaml]
+----
```

Comment:
both the Secret and ConfigMap are in the generated `keycloak-ispn.yaml` and can be imported.

Reply:
We are not pointing to the `keycloak-ispn.yaml` in our docs, it was merely done for Automation. If you think we should point the users to the generated yaml file, we need to change our workflow a little bit, let me know what you think, thats the reason I was calling them out separately.

Reply:
yes we are, here: https://github.com/keycloak/keycloak-benchmark/pull/566/files#diff-cd3c22227b856c0da2469d4ee3bb053ffd9c9ba5f9d3b1c68b24f0af42cfcad3R54

Reply:
@ahus1 we might need your input here, do you think we should point to the `keycloak-ispn.yaml` and display the Secret and ConfigMap together instead of separate blocks of yaml, save the users some time to re-arrange the pieces ?